pkgs/data/fonts/u001/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "U001";
+  pname = "u001";
+  version = "unstable-2022-02-05"; # date in the zip file, actual creation date unknown
+
+  src = fetchurl {
+    # this URL had Condensed versions that seemed missing elsewhere
+    url = "https://www.cufonfonts.com/download/font-family/4425073b3af78b6b234e706084cddf6b/u001";
+    sha256 = "sha256-iamnUgS30igtgesFD10wwNKCgpiM++DgiMoNjCkVpv4=";
+  };
+
+  nativeBuildInputs = [ unzip ];
+
+  dontBuild = true;
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    unzip $src
+
+    runHook postUnpack
+  '';
+
+  # We’ll adjust the nonstandard naming convention here
+  installPhase = ''
+    runHook preInstall
+
+    TTF=$out/share/fonts/truetype
+
+    mkdir -p $TTF
+
+    cp u001-reg.ttf $TTF/U001-Regular.ttf
+    cp u001-ita.ttf $TTF/U001-Italic.ttf
+    cp u001-bol.ttf $TTF/U001-Bold.ttf
+    cp u001-bolita.ttf $TTF/U001-BoldItalic.ttf
+    cp u001con-reg.ttf $TTF/U001Condensed-Regular.ttf
+    cp u001con-ita.ttf $TTF/U001Condensed-Italic.ttf
+    cp u001con-bol.ttf $TTF/U001Condensed-Bold.ttf
+    cp u001con-bolita.ttf $TTF/U001Condensed-BoldItalic.ttf
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A Univers-like typeface that comes with GhostPDL made by URW++";
+    license = licenses.aladdin;
```

Comment:
Where did you find what license the font has? I can only see this on their [homepage](https://www.cufonfonts.com/font/u001):

> License
> (URW)++,Copyright 2000 by (URW)++ Design & Development

Reply:
https://fontlibrary.org/en/font/u001 

Shows Aladdin but is only the Condensed fonts (it's also the page referenced in the Wikipedia article)

https://en.wikipedia.org/wiki/Univers#URW_Classic_Sans

The Wikipedia section mentions it's a part of GhostPDL.

https://en.wikipedia.org/wiki/Ghostscript#Free_fonts

Shows U001 under the list of fonts under AFPL

Reply:
I omitted a `homepage` because it doesn't seem to really have a "home" anymore and has just been hosted in some places (some of which lacking the correct license line, though the font from cufonts does support 4 styles in both regular and codensed).

Reply:
Moved to the https://fontlibrary.org/en/font/u001 option